homeassistant/components/sensor/tellduslive.py

```diff
@@ -127,3 +127,8 @@ def device_class(self):
         """Return the device class."""
         return SENSOR_TYPES[self._type][3] \
             if self._type in SENSOR_TYPES else None
+
+    @property
+    def unique_id(self) -> str:
+        """Return a unique ID."""
+        return "tellduslive-{}".format("-".join(self._id[:-1]))
```

Comment:
No need to prefix with platform name. 

Why chop off last element from the ID ?

Reply:
Because the tellduslive library returns `[id, type, value]` as id 

https://github.com/home-assistant/home-assistant/blob/9d7b1fc3a75fb02e92410b1abdd4f133a10d5963/homeassistant/components/sensor/tellduslive.py#L70